### ISSUE TITLE
Incorrect depth occlusion after combining scene model with nerf

### ISSUE THREAD

**Issue by ydi-mua** [NVlabs/instant-ngp#1403]
I import the nerf trained file (. ingp) into the scene and call instantngp
Corresponding interface, after synchronizing the camera, obtain real-time depth of nerf and find that the depth is incorrect

https://github.com/NVlabs/instant-ngp/assets/105626420/3251dedc-3a6e-41d2-b563-9fde2f0f2a58

**Comment by ydi-mua** [comment]
resolved

**Comment by zzy729425207** [comment]
Hi, I want to get depth image using testbed.render, but I get white image which I think is incorrect. How can I get right Depth image?

**Comment by ydi-mua** [comment]
> Hi, I want to get depth image using testbed.render, but I get white image which I think is incorrect. How can I get right Depth image?

You need to modify the source code, because the depth distance is forced to clamp(z, n, f) in the source code.so the farther you are away, the depth value becomes 1 (white (1, 1, 1, 1)).
![image](https://github.com/NVlabs/instant-ngp/assets/105626420/a9db4cfa-9cfb-4abe-a3a0-757a23de2134)

Recalculate the depth value yourself according to your needs

**Comment by YuFengUofR** [comment]
Hi, I tried to use depth images to align two images. But no matter how I use the transformation matrix provided by "transform.json", I always obtain incorrect results. This is what I did:

1. I first generated depth maps.
2.  then I used depth maps and camera parameters like "fl_x", "fl_y", "cx" and "cy" to generate two 3D point clouds from two images. 
3. next, I used the transformation matrix to multiply with point clouds to transform the point clouds to the same coordinate system. (BTW, I also tried to use the inversed transformation matrix, it didn't work also.)

did I do anything incorrectly?

Thanks!
